android/modules/ui/src/java/ti/modules/titanium/ui/android/DrawerLayoutProxy.java

```diff
@@ -220,6 +224,27 @@ public void setDrawerLockMode(Object arg)
 		setPropertyAndFire(TiC.PROPERTY_DRAWER_LOCK_MODE, arg);
 	}
 
+	// clang-format off
+	@Kroll.method
+	@Kroll.getProperty
+	public int getDrawerGravity()
+	// clang-format on
+	{
+		if (hasProperty(TiC.PROPERTY_DRAWER_GRAVITY)) {
+			return (Integer) getProperty(TiC.PROPERTY_DRAWER_GRAVITY);
+		}
+		return GRAVITY_BOTH;
```

Comment:
Wouldn't it make more sense to have a `GRAVITY_NONE` equivalent to `Gravity.None` which is returned when no gravity is set?